lib/bambora/rest/client.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Bambora
+  module Rest
```

Comment:
I don't have a preference, but am commenting on the inconsistency of using `REST` vs `Rest` compared with `WWW` -> `Www` and `XML` -> `Xml`. Looks like everything is explicitly required so I think it's all good, but maybe this should be REST? I assume this has to do with constants?

Reply:
Ha, I thought about this. I chose `Rest` because it should technically be `ReST`, which I think looks weird.

Reply:
Funny though, that I called it `REST` in one of my commits. I guess I don't have a strong opinion on this other than I wish the E stood for something.